lua/gatekeeper.lua

```diff
@@ -99,6 +99,11 @@ local ffi = require("ffi")
 -- Structs
 ffi.cdef[[
 
+struct constants {
+	static const uint32_t BONDING_MODE_ROUND_ROBIN = 0;
+	static const uint32_t BONDING_MODE_8023AD = 4;
```

Comment:
1. I didn't include the other modes because I haven't seen a need for them in Gatekeeper, and I didn't want to add more risk in adding them without purpose. Looking at the documentation, it seems only the LACP mode requires further application restrictions -- the rest should be naturally supported. Should I add the others?

2. Thanks, will do.